src/docfx/cli/New.cs

```diff
@@ -83,6 +83,15 @@ private static bool CreateFromTemplate(string templateName, CommandLineOptions o
             }
             else
             {
+                var directory = Path.GetDirectoryName(targetFullPath) ?? ".";
+                if (!Directory.Exists(directory))
+                {
+                    Directory.CreateDirectory(directory);
+                    if (TestQuirks.Initializable ?? false)
```

Comment:
Would it be better to add this param to `docfx new`?

Reply:
I don't think so. Initializing a git directory is not required when creating a new docset. This is only for test usage